src/com/google/javascript/jscomp/deps/ModuleLoader.java

```diff
@@ -298,6 +299,28 @@ static String normalize(String path, Iterable<String> moduleRootPaths) {
     return path;
   }
 
+  public static String relativePathFrom(String fromUriPath, String toUriPath) {
+    Path fromPath = Paths.get(fromUriPath);
+    Path toPath = Paths.get(toUriPath);
+    Path fromFolder = fromPath.getParent();
+
+    // if the from URIs are simple names without paths, they are in the same folder
+    // example: m0.js
+    if (fromFolder == null && toPath.getParent() == null) {
+      return "./" + toUriPath;
+    } else if (fromFolder == null && toUriPath.startsWith(".") || toPath.startsWith("/")) {
```

Comment:
our internal checks want grouping parentheses to make operator precedence explicit here.

Reply:
Rightly so - added.